testing/describes.js

```diff
@@ -101,6 +101,7 @@ import {
 import {createElementWithAttributes} from '../src/dom';
 import {addParamsToUrl} from '../src/url';
 import {cssText} from '../build/css';
+import {CSS} from '../build/amp-ad-0.1.css.js';
```

Comment:
@bradfrizzell FYI, this change seems to have broken `gulp test --unit --files <file>` when run from a clean state. I've opened #11613 to track this.

Reply:
Noted. I will try to get a fix out for this ASAP.

Reply:
I have a fix :) Sending you a PR.

Reply:
Haha perfect! Thanks Raghu. Sorry about the headache!

Reply:
All is well!

Reply:
Fix is ready for review in #11615.